weave/Scripts/MenuControllers/StartScreen.cs

```diff
@@ -175,24 +210,26 @@ private void OnStartButtonPressed()
     private void ExpandButtons()
     {
         _playButton.Text = "PLAY";
-        _optionsButton.Text = "OPTIONS";
+        _playButton.CustomMinimumSize = new Vector2(200, 0);
+
         _quitButton.Text = "QUIT";
-        _playButton.CustomMinimumSize = new(200, 0);
-        _optionsButton.CustomMinimumSize = new(200, 0);
-        _quitButton.CustomMinimumSize = new(200, 0);
+        _quitButton.CustomMinimumSize = new Vector2(200, 0);
     }
 
     private void CollapseButtons()
     {
+        if (_buttonContainer.GetGlobalRect().HasPoint(GetGlobalMousePosition()))
```

Comment:
Whats the purpose of this? :)

Reply:
Without it the buttons collapse when moving the mouse between them (since we're exiting one and entering the other) so we only collapse them when we actually leave the container they're in